diff --git a/hw/obi_uart/obi_uart.sv b/hw/obi_uart/obi_uart.sv
index 8d10da6..a727b6b 100644
--- a/hw/obi_uart/obi_uart.sv
+++ b/hw/obi_uart/obi_uart.sv
@@ -9,12 +9,12 @@
 // `include "common_cells/registers.svh"
 
 module obi_uart #(
-  /// The OBI configuration connected to this peripheral.
-  parameter obi_pkg::obi_cfg_t ObiCfg = obi_pkg::ObiDefaultConfig, // SbrObiCfg
-  /// OBI request type
-  parameter type obi_req_t = logic,
-  /// OBI response type
-  parameter type obi_rsp_t = logic
+  // /// The OBI configuration connected to this peripheral.
+  // parameter obi_pkg::obi_cfg_t ObiCfg = obi_pkg::ObiDefaultConfig, // SbrObiCfg
+  // /// OBI request type
+  // parameter type obi_req_t = logic,
+  // /// OBI response type
+  // parameter type obi_rsp_t = logic
 ) (
   input logic      clk_i,  // Primary input clock
   input logic      rst_ni, // Asynchronous active-low reset
diff --git a/hw/obi_uart/obi_uart_register.sv b/hw/obi_uart/obi_uart_register.sv
index 349638f..20859aa 100644
--- a/hw/obi_uart/obi_uart_register.sv
+++ b/hw/obi_uart/obi_uart_register.sv
@@ -9,12 +9,12 @@
 // `include "common_cells/registers.svh"
 
 module obi_uart_register import obi_uart_pkg::*; #(
-  /// The OBI configuration connected to this peripheral.
-  parameter obi_pkg::obi_cfg_t ObiCfg = obi_pkg::ObiDefaultConfig, // SbrObiCfg
-  /// OBI request type
-  parameter type obi_req_t = logic,
-  /// OBI response type
-  parameter type obi_rsp_t = logic
+  // /// The OBI configuration connected to this peripheral.
+  // parameter obi_pkg::obi_cfg_t ObiCfg = obi_pkg::ObiDefaultConfig, // SbrObiCfg
+  // /// OBI request type
+  // parameter type obi_req_t = logic,
+  // /// OBI response type
+  // parameter type obi_rsp_t = logic
 ) (
   input logic clk_i,
   input logic rst_ni,
