diff --git a/tmrg_files/obi_uartTMR.sv b/tmrg_files/obi_uartTMR.sv
index 8d9721d..50513aa 100644
--- a/tmrg_files/obi_uartTMR.sv
+++ b/tmrg_files/obi_uartTMR.sv
@@ -26,7 +26,14 @@
  *                                                                                                  *
  ****************************************************************************************************/
 
-module obi_uartTMR (
+module obi_uartTMR #(
+  /// The OBI configuration connected to this peripheral.
+  parameter obi_pkg::obi_cfg_t ObiCfg = obi_pkg::ObiDefaultConfig, // SbrObiCfg
+  /// OBI request type
+  parameter type obi_req_t = logic,
+  /// OBI response type
+  parameter type obi_rsp_t = logic
+) (
   input logic clk_i,
   input logic rst_ni,
   input obi_req_t obi_req_iA,
diff --git a/tmrg_files/obi_uart_registerTMR.sv b/tmrg_files/obi_uart_registerTMR.sv
index 9beac00..0aca1f1 100644
--- a/tmrg_files/obi_uart_registerTMR.sv
+++ b/tmrg_files/obi_uart_registerTMR.sv
@@ -26,7 +26,14 @@
  *                                                                                                  *
  ****************************************************************************************************/
 
-module obi_uart_registerTMR import obi_uart_pkg::*; (
+module obi_uart_registerTMR import obi_uart_pkg::*; #(
+  /// The OBI configuration connected to this peripheral.
+  parameter obi_pkg::obi_cfg_t ObiCfg = obi_pkg::ObiDefaultConfig, // SbrObiCfg
+  /// OBI request type
+  parameter type obi_req_t = logic,
+  /// OBI response type
+  parameter type obi_rsp_t = logic
+) (
   input logic clk_i,
   input logic rst_ni,
   input obi_req_t obi_req_iA,
